services/ui-src/src/utils/reports/entities.ts

```diff
@@ -1,9 +1,10 @@
 import { EntityShape, OverlayModalEntityTypes, AnyObject } from "types";
 
 const getRadioValue = (entity: EntityShape | undefined, label: string) => {
+  const radioLabelValue = entity?.[label]?.[0].value;
   return entity?.[label]?.[0].value !== "Other, specify"
-    ? entity?.[label]?.[0].value
-    : entity?.[label + "-otherText"];
+    ? radioLabelValue
```

Comment:
you could replace`entity?.[label]?.[0].value` on line 5 with `radioLabelValue` as well

Reply:
Ah good catch, literally the same value lol